[2025-07-29T10:31:49.004882] ๐ด ููููุฏ: kernel/kernel.bin
[2025-07-29T10:31:49.006580] ๐ด ููููุฏ: boot/grub/grub.cfg
[2025-07-29T10:31:49.008151] ๐ด ููููุฏ: init/system_init.py
[2025-07-29T10:31:49.008714] ๐ด ููููุฏ: obeyx_core/obeyx_brain.py
[2025-07-29T10:31:49.010229] ูุดู ุงูุฅุฑุณุงู ุงููุฑูุฒู: [Errno 111] Connection refused
[2025-07-29T10:31:49.012081] โ ูุง ูููู ุชุฑููู ุงูููู ุงูููููุฏ ุชููุงุฆููุง: kernel/kernel.bin
[2025-07-29T10:31:49.012871] โ ูุง ูููู ุชุฑููู ุงูููู ุงูููููุฏ ุชููุงุฆููุง: boot/grub/grub.cfg
[2025-07-29T10:31:49.013820] โ ูุง ูููู ุชุฑููู ุงูููู ุงูููููุฏ ุชููุงุฆููุง: init/system_init.py
[2025-07-29T10:31:49.014353] โ ูุง ูููู ุชุฑููู ุงูููู ุงูููููุฏ ุชููุงุฆููุง: obeyx_core/obeyx_brain.py
